oauthenticator/google.py

```diff
@@ -105,6 +105,21 @@ def _userdata_url_default(self):
         """,
     )
 
+    allow_nested_groups = Bool(
+        config=True,
+        help="""
+        Allow members of nested Google groups to sign in and/or administer
+        TLJH. If `True` the authenticator will recursively check a user's 
+        group memberships and allow sign in and/or grant admin rights to
+        a user that is a member of a group that has nested membership in
+        any group in `allowed_google_groups` or `admin_google_groups`.
+
+        Use of this requires configuration of `gsuite_administrator`, and
```

Comment:
Lets trim away some docs here, letting the groups config describe these requirements about service account keys etc, and settle by saying that this is a config that makes the config options admin/allowed groups accept the listed groups' nested groups as well.

Or, is this an additional requirement that isnt a requirement on admin groups / allowed groups?